File: Serene/Serene.Script/Meeting/MeetingDecision/MeetingDecisionGrid.cs

namespace Serene.Meeting
{
    using jQueryApi;
    using Serenity;
    using System;
    using System.Collections.Generic;
    using System.Runtime.CompilerServices;

    [ColumnsKey("Meeting.MeetingDecision"), IdProperty("MeetingDecisionId"), NameProperty("Description"), IsActiveProperty("IsActive")]
    [DialogType(typeof(MeetingDecisionDialog)), LocalTextPrefix("Meeting.MeetingDecision"), Service("Meeting/MeetingDecision")]
    public class MeetingDecisionGrid : EntityGrid<MeetingDecisionRow>
    {
        public MeetingDecisionGrid(jQueryObject container)
            : base(container)
        {
        }
    }

    // Please remove this partial class or the first line below, after you run ScriptContexts.tt
    [Imported, Serializable, PreserveMemberCase] 
    public partial class MeetingDecisionRow
    {
    }
}

File: Serene/Serene.Web/Modules/Meeting/MeetingDecision/MeetingDecisionGrid.ts

namespace Serene.Meeting {
    
    @Serenity.Decorators.registerClass()
    export class MeetingDecisionGrid extends Serenity.EntityGrid<MeetingDecisionRow, any> {
        protected getColumnsKey() { return 'Meeting.MeetingDecision'; }
        protected getDialogType() { return MeetingDecisionDialog; }
        protected getIdProperty() { return MeetingDecisionRow.idProperty; }
        protected getLocalTextPrefix() { return MeetingDecisionRow.localTextPrefix; }
        protected getService() { return MeetingDecisionService.baseUrl; }

        constructor(container: JQuery) {
            super(container);
        }
    }
}